5/Dockerfile.fedora

```diff
@@ -0,0 +1,39 @@
+# requires https://src.fedoraproject.org/container/s2i-core/pull-request/1
+FROM candidate-registry.fedoraproject.org/f27/s2i-core:latest
+
+EXPOSE 8080
+EXPOSE 8443
+
+ENV SUMMARY="Platform for running Varnish or building Varnish-based application" \
+    DESCRIPTION="Varnish available as docker container is a base platform for \
+running Varnish server or building Varnish-based application. \
+Varnish Cache stores web pages in memory so web servers don't have to create \
+the same web page over and over again. Varnish Cache serves pages much faster \
+than any application server; giving the website a significant speed up." \
+    VARNISH_CONFIGURATION_PATH=/etc/varnish
+
+LABEL summary="$SUMMARY" \
+      description="$DESCRIPTION" \
+      io.k8s.description="$DESCRIPTION" \
+      io.k8s.display-name="Varnish 5" \
+      io.openshift.expose-services="8080:http,8443:https" \
+      io.openshift.tags="builder,varnish" \
+      name="f27/varnish" \
+      version="0"
```

Comment:
According to Fedora guidelines `release`, `architecture`, `maintainer` and `usage/run` labels are missing. Having right spec in github will help when someone will try to get the image into Fedora (pass container review).

Is inhering them from base image enough?

Reply:
You're right! I'm adding release, maintainer and usage; architecture should be inherited.